### ISSUE TITLE
Revisions to OBO Foundry page dropdown menu - Participate & About

### ISSUE THREAD

**Issue by nataled** [OBOFoundry/OBOFoundry.github.io#198]
Per agenda item 3 in https://docs.google.com/document/d/1d3c9NOv-zKKHhOpTdafamxHIkljy_Z8vcKzG-jt_zSc/edit#

1) Move "News" from Participate to About

2) Move "Twitter feed (coming soon) to About

3) Add "Register my ontology" to the Participate dropdown menu underneath "Join the mailing list" and link the new item to http://obofoundry.org/docs/Policy_for_OBO_namespace_and_associated_PURL_requests.html

4) Add "Review my ontology”  to the Participate dropdown menu underneath "Register my ontology" and link the new item to http://obofoundry.org/docs/ReviewProcessGuidelines.html

**Comment by cmungall** [comment]
1. +1
 2. -1. Unless someone in outreach wants to take the initiative and set up a twitter account, kill this item
 3. +1
 4. +1

**Comment by nataled** [comment]
By "kill this item" I assume you mean to remove Twitter from the menu altogether. I agree (we were only moving things around, and did not discuss removing anything).

**Comment by cmungall** [comment]
Yes, kill the menu item

**Comment by cstoeckert** [comment]
Discussed Twitter feed on Feb. 9 call and agree to "kill".

**Comment by nlharris** [comment]
it looks like things 1,3,4 are not done. do we want them done? if so, i could try to do it.

**Comment by nlharris** [comment]
> 1. Move "News" from Participate to About
Since there hasn't been any news posted since 2017, I removed News from the menu in #1297, so this item is now obsolete.

> 3. Add "Register my ontology" to the Participate dropdown menu underneath "Join the mailing list" and link the new item to http://obofoundry.org/docs/Policy_for_OBO_namespace_and_associated_PURL_requests.html

**Comment by nicolevasilevsky** [comment]
3. On this PR https://github.com/OBOFoundry/OBOFoundry.github.io/pull/1301, it will (_should?_) change the link to faq/how-do-i-register-my-ontology.html, so I think that will take care of this once the PR is merged.

4. Re: the ReviewProcessGuidelines page:
- do we want add spaces to the title, instead of CamelCase?
- under the 'How to request an ontology review', should we also have a step to email the OBO list? 
- These same instructions are now in 3 places, right? This is redundant with the FAQ page (http://obofoundry.org/faq/how-do-i-submit-my-ontology.html) and the membership requirements page (http://obofoundry.org/docs/Policy_for_OBO_namespace_and_associated_PURL_requests.html) -

**Comment by nlharris** [comment]
@nicolevasilevsky can we assign you to this?

**Comment by nicolevasilevsky** [comment]
sure!

**Comment by nicolevasilevsky** [comment]
Looking at this now - 
I don't see a 'news' page under participate, does this still exist?
@nataled

**Comment by nataled** [comment]
Per @nlharris comment of Sept 29th, that was removed from the menu completely.

**Comment by nataled** [comment]
@nicolevasilevsky the Twitter feed item was also slated to be removed completely.

**Comment by nlharris** [comment]
yeah, we had a discussion about this, and since we're not posting any news or tweeting, it seemed pointless to have those on the menu. if we get new energy (maybe from you, @nicolevasilevsky??) to post/tweet again, we can re-add them. it's not that we never have anything newsworthy, but that no one had the time/energy to do the social media.

**Comment by nicolevasilevsky** [comment]
@nataled @nlharris - thanks, I missed that! I removed the twitter and news feed (just commented them out) on PR #1381

**Comment by nicolevasilevsky** [comment]
@nlharris do we still want to step # 4 above?

**Comment by nlharris** [comment]
> do we want add spaces to the title, instead of CamelCase?

although clearly that would make it more readable, i am leery of changing it, given the issues we've encountered with changing page names. maybe changing the title wouldn't affect the URL, so it would all be fine, but i'm not sure.

> under the 'How to request an ontology review', should we also have a step to email the OBO list?

not sure. seems like submitting a ticket should be good enough; we can internally start a discussion on the obo list if needed.

> These same instructions are now in 3 places, right? This is redundant with the FAQ page (http://obofoundry.org/faq/how-do-i-submit-my-ontology.html) and the membership requirements page (http://obofoundry.org/docs/Policy_for_OBO_namespace_and_associated_PURL_requests.html)

there is some overlap, yes, but since this is specifically about requesting a review, i think it's worth having that page, even if the info is partly redundant with those other pages.

however, i'm still not sure if we want to actively encourage people to request ontology reviews, given our limited person-time to respond to such requests.

**Comment by nlharris** [comment]
i'm going to close this -- see my new ticket #1930
